storage/file.go

```diff
@@ -92,40 +292,176 @@ func pathForFileShare(name string) string {
 // See https://msdn.microsoft.com/en-us/library/azure/dd179352.aspx
```

Comment:
It appears that all of the links are redirecting to the new docs location.  Let's do a scrub of these in a separate PR.